### ISSUE TITLE
[Question] Adding Wildcard Certificates

### ISSUE THREAD

**Issue by PBHDK** [shibayan/appservice-acmebot#234]
Hi there, 

Sorry, if this is a super obvious question, but how do I add a wildcard certificate? I have all the correct role assignments setup, but the func-.../add-certificate screen looks just as before? 

Many thanks!

**Comment by shibayan** [comment]
The issuance of a wildcard certificate is the same as a regular certificate. However, if the conditions are not met, a runtime error will occur.

As mentioned in the README, the following conditions are required

- The App Service has added a wildcard domain
- That domain is being delegated to Azure DNS

**Comment by PBHDK** [comment]
Cheers for getting back to me! 

Ok, let me rephrase a bit. 

So, I've got my site running in an app service. I want all traffic to go to https://example.com, i.e. all let's say five subdomains (www, blog, music, ... ) will be redirected to https://example.com. In order to make this possible, from my understanding, I'll have to add all of the subdomains to the app service as well. So, my custom domains for my app service will look something like this:
- example.azurewebsites.com
- www.example.com
- blog.example.com
- music.example.com
... 

In the future, I'll probably want to add more subdomains without having to reissue a certificate. So, I'd like to obtain a root cert for all the subdomains and have it auto-renew with the appservice-acmebot. How would I do this?

**Comment by shibayan** [comment]
Okay, I understand what you want to achieve.

App Service Acmebot requires custom domain assignments, so in your case you'll be using a SAN certificate instead of a wildcard certificate.

So I recommend using Key Vault Acmebot.
https://github.com/shibayan/keyvault-acmebot

This app allows you to obtain a certificate with only Key Vault and Azure DNS freely, so if you create a wildcard certificate first, you can use it freely later.

**Comment by PBHDK** [comment]
YES! This did exactly what I needed! Thank you so muche - loving this setup! :)
